src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/ApplicationGateway/New-ArmApplicationGatewayResource.ps1

```diff
@@ -17,6 +17,11 @@ function New-ArmApplicationGatewayResource {
         [ValidateSet("Prevention", "Detection")]
         [string]
         $FirewallMode = "Prevention"
+        [bool]
+        $RequestBodyCheck = $false,
+        [ValidateRange(8, 128)]
+        [int]
+        $MaxRequestBodySize = 128
```

Comment:
Add unit in the var name
Is it bytes kb MB?
I.e. MaxRequestBodySizeInKb

Reply:
kb